streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -714,63 +618,27 @@ public void shouldAllowSettingProducerEnableIdempotenceIfEosDisabled() {
         assertThat(producerConfigs.get(ProducerConfig.ENABLE_IDEMPOTENCE_CONFIG), equalTo(false));
     }
 
-    @SuppressWarnings("deprecation")
-    @Test
-    public void shouldSetDifferentDefaultsIfEosAlphaEnabled() {
-        props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, EXACTLY_ONCE);
-        shouldSetDifferentDefaultsIfEosEnabled();
-    }
-
-    @SuppressWarnings("deprecation")
-    @Test
-    public void shouldSetDifferentDefaultsIfEosBetaEnabled() {
-        props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, EXACTLY_ONCE_BETA);
-        shouldSetDifferentDefaultsIfEosEnabled();
-    }
-
     @Test
     public void shouldSetDifferentDefaultsIfEosV2Enabled() {
         props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, EXACTLY_ONCE_V2);
-        shouldSetDifferentDefaultsIfEosEnabled();
-    }
-
-    private void shouldSetDifferentDefaultsIfEosEnabled() {
         final StreamsConfig streamsConfig = new StreamsConfig(props);
 
         final Map<String, Object> consumerConfigs = streamsConfig.getMainConsumerConfigs(groupId, clientId, threadIdx);
         final Map<String, Object> producerConfigs = streamsConfig.getProducerConfigs(clientId);
 
         assertThat(
-            consumerConfigs.get(ConsumerConfig.ISOLATION_LEVEL_CONFIG),
-            equalTo(READ_COMMITTED.name().toLowerCase(Locale.ROOT))
+                consumerConfigs.get(ConsumerConfig.ISOLATION_LEVEL_CONFIG),
+                equalTo(READ_COMMITTED.name().toLowerCase(Locale.ROOT))
```

Comment:
Also here 4 spaces are enough:
```suggestion
            consumerConfigs.get(ConsumerConfig.ISOLATION_LEVEL_CONFIG),
            equalTo(READ_COMMITTED.name().toLowerCase(Locale.ROOT))
```

Reply:
Done